kiosk/src/fonts/Press_Start_2P/OFL.txt

```diff
@@ -0,0 +1,93 @@
+Copyright 2012 The Press Start 2P Project Authors (cody@zone38.net), with Reserved Font Name "Press Start 2P".
```

Comment:
^^ I'll defer to abhijith / richard on this as I don't know if we need to do anything when checking in fonts, the license seems to be permissive at a glance so should be fine~